### ISSUE TITLE
Typescipt type

### ISSUE THREAD

**Issue by malekim** [intlify/vue-i18n-next#503]
What is the typescript type for createI18n() instance? If I use simple VueI18n, then I've got an error: 

> Type 'I18n<LocaleMessages<VueMessageType>, unknown, unknown, true>' is missing the following properties from type 'VueI18n<{}, {}, {}>': id, locale, fallbackLocale, availableLocales, and 32 more.

```
const messages: LocaleMessages<VueMessageType> = {
  eng: eng,
  ger: ger,
}

const i18n: VueI18n = createI18n({
  locale: 'eng',
  fallbackLocale: 'eng',
  messages: messages,
})
```

**Comment by kazupon** [comment]
Thank you for your feedback!

I'm working on typescript improvement at [this PR](https://github.com/intlify/vue-i18n-next/pull/489)
This improvement will be released v9.2
please wait.

**Comment by malekim** [comment]
Ok, great, thank you! If anyone get into that issue through google, just use type 'any' for now.
